tictactoe.py

```diff
@@ -0,0 +1,113 @@
+def game() :
+    #User input for the desired symbol of the player
+    player1 = input("Player 1, enter your desired symbol. Press Enter for default\n")
+    #Default symbol for Player 1
+    if player1 == "":
+        player1 = "X"
+        print("Player 1: X")
+    #User input for the desired symbol of the player
+    player2 = input("Player 2, enter your desired symbol. Press Enter for default\n")
+    #Default symbol for Player 2
+    if player2 == player1:
+        player2 = input("Invalid symbol entered, please enter a different symbol. Press Enter for default\n")
+    if player2 == "":
+        player2 = "O"
+        print("Player 2: O")
+    #Integer that tracks the number of plays that have occurred
+    count = 0;
+    #Boolean that tracks which users turn it is
+    flag = True
+    #While loop that iterates 9 times (limit based on size of board)
+    while count < 9:
+        #Prints the current board for every turn
+        printBoard()
+        #Conditional for Player 1's turn
+        if flag:
+            move = input("Player 1's turn, choose a spot to place your symbol\n")
+            #Checks if the desired space is empty
+            if move.isdigit() == False or int(move) > 9 or int(move) < 0:
+                print("Invalid selection, please choose a correct space")
+                continue
+            elif board[move] == ' ':
+                #Places player 1's symbol in the correct spot
+                board[move] = player1
+                #Increments the number of plays that have occurred
+                count += 1
+            else:
+                print("That spot is already filled. Choose a spot to place your symbol\n")
+                continue
+            if count >= 3 and checkWin(player1):
+                print("Player 1 Wins!\n")
+                print("\n" + "Final Board:")
+                printBoard()
+                print("\n")
+                break
+            flag = False
+        else:
+            move = input("Player 2's turn, choose a spot to place your symbol\n")
+            if move.isdigit() == False or int(move) > 9 or int(move) < 0:
+                print("Invalid selection, please choose a correct space")
+                continue
+            elif board[move] == ' ':
+                board[move] = player2
+                count += 1
+            else:
+                print("That spot is already filled. Choose a spot to place your symbol\n")
+                continue
+            if count >= 3 and checkWin(player2):
+                print("Player 2 Wins!\n")
+                print("\n" + "Final Board:")
+                printBoard()
+                print("\n")
+                break
+            flag = True
+    if count == 9 and not checkWin(player2) or not checkWin(player1):
+        print("Game Over! It's a tie!\n")
+    restart = input("Do you want to play again? (Y/N)\n")
+    if restart == "y" or restart == "Y":
+        for key in board_keys:
+            board[key] = " "
```

Comment:
I would probably make a createClearBoard method that you call at the beginning of each game function to give you a clean board